server/package.json

```diff
@@ -87,14 +87,19 @@
     "joi": "^14.3.1",
     "jsonwebtoken": "^8.4.0",
     "lodash.clonedeep": "^4.5.0",
+    "mock-http-server": "^1.4.2",
     "node-nlp": "^3.0.3",
     "node-schedule": "^1.3.2",
     "node-webcrypto-ossl": "^1.0.48",
+    "nyc": "^15.1.0",
+    "oauth2-mock-server": "^1.5.1",
```

Comment:
If you add dev-dependencies, please add them to the devDependencies not the dependencies

Why do you need nyc btw?

Reply:
I remove nyc (it's a mistake whenI try to execute coverage in my computer, I don't understand ho I can run it, I always push PR to have the coverage ...)

oauth2-mock-server is used in oauth2 test, I put it in devDependencies

Reply:
I remove db file

Reply:
If you have no withings account I can give you my client & secret id for test

Reply:
> I remove nyc (it's a mistake whenI try to execute coverage in my computer, I don't understand ho I can run it, I always push PR to have the coverage ...)

You can execute it with `npm run coverage` or `./node_modules/.bin/nyc --reporter=html npm run test-service --service=withings`
The second command will generate a `coverage` folder with report inside (`index.html`)

Reply:
many thanks